packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx

```diff
@@ -0,0 +1,300 @@
+                {isRefetching ? (
+                  <OsdsSpinner inline={true} size={ODS_SPINNER_SIZE.md} />
+                ) : (
+                  <OsdsButton
+                    slot="reset-trigger"
+                    size={ODS_BUTTON_SIZE.sm}
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    variant={ODS_BUTTON_VARIANT.stroked}
+                    onClick={handleRefresh}
+                    {...(isFetching && { disabled: true })}
+                  >
+                    <span slot="start" className="flex items-center mr-0">
+                      <OsdsIcon
+                        name={ODS_ICON_NAME.REFRESH}
+                        size={ODS_ICON_SIZE.sm}
+                        color={ODS_THEME_COLOR_INTENT.primary}
+                      />
+                    </span>
+                  </OsdsButton>
+                )}
+              </div>
+              <SearchBar
+                onSearchSubmit={handleSearchSubmit}
+                isDisabled={filters.length > 0 || isFetching}
+              />
+              <OsdsPopover ref={filterPopoverRef}>
+                <OsdsButton
+                  slot="popover-trigger"
+                  size={ODS_BUTTON_SIZE.sm}
+                  color={ODS_THEME_COLOR_INTENT.primary}
+                  variant={ODS_BUTTON_VARIANT.stroked}
+                  {...((filters.length > 0 || isFetching) && {
+                    disabled: true,
+                  })}
+                >
+                  <span slot="start" className="flex items-center mr-0">
+                    <OsdsIcon
+                      name={ODS_ICON_NAME.FILTER}
+                      size={ODS_ICON_SIZE.sm}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                    />
+                    <span>{t('common:pci_instances_common_filter')}</span>
+                  </span>
+                </OsdsButton>
+                <OsdsPopoverContent>
+                  <FilterAdd
+                    columns={filterColumns}
+                    onAddFilter={(addedFilter, column) => {
+                      addFilter({
+                        ...addedFilter,
+                        label: column.id,
+                      });
+                      filterPopoverRef.current?.closeSurface();
+                    }}
+                  />
+                </OsdsPopoverContent>
+              </OsdsPopover>
+            </div>
+          </div>
+          <div className="my-5">
+            <FilterList filters={filters} onRemoveFilter={removeFilter} />
+          </div>
+          <DatagridComponent
+            sorting={sorting}
+            filters={filters}
+            onRefresh={handleRefresh}
+            onSortChange={setSorting}
+          />
+          {isFetchingNextPage && (
+            <div className="mt-5">
+              <Spinner />
+            </div>
+          )}
+        </div>
+      </PageLayout>
+      {!isFetching && <Outlet />}
```

Comment:
Why is this used for ? I think outlet should be displayed even if you are fetching (or maybe isPending is more appropriate), the children should chose to render or not using suspense or loading